boa/interpret.py

```diff
@@ -24,6 +25,37 @@
 _disk_cache = None
 
 
+class BoaImporter:
+    def find_module(self, module_name, package_path):
+        # Return a loader
+        return self
+
+    def load_module(self, fullname):
+        # Return a module
+        if fullname in sys.modules:
+            return sys.modules[fullname]
+
+        path = Path(fullname.replace(".", "/")).with_suffix(".vy")
+        for prefix in sys.path:
+            to_try = Path(prefix) / path
+            try:
+                ret = load_partial(to_try)
+                # comply with PEP-302:
+                ret.__name__ = to_try.name
+                ret.__file__ = str(to_try)
+                break
+            except (FileNotFoundError, NotADirectoryError):
+                pass
+        else:
+            raise ImportError(fullname)
+
+        sys.modules[fullname] = ret
+        return ret
+
+
+sys.meta_path.append(BoaImporter())  # type: ignore
```

Comment:
`BoaImporter` should implement [MetaPathFinder](https://docs.python.org/3/library/importlib.html#importlib.abc.MetaPathFinder)
```suggestion
sys.meta_path.append(BoaImporter())
```